src/models/reranking.rs

```diff
@@ -4,6 +4,8 @@ pub enum RerankerModel {
     BGERerankerBase,
     // jinaai/jina-reranker-v1-turbo-en
     JINARerankerV1TurboEn,
+    // jinaai/jina-reranker-v2-base-multilingual
+    JINARerankerV2BaseMultiligual,
```

Comment:
Yikes. Just noticed.
These 2 should've been doc comments like the first.
///

Reply:
Yes. I was simply following the previous JINA model setup and didn't notice this. I look forward to seeing the changes in the next release.